content/monte-carlo-on-ec2-spot-fleet/lab4.md

```diff
@@ -45,7 +46,7 @@ $(aws ecr get-login --region $REGION --no-include-email)
 
 1. Create a repository named **monte-carlo-workshop**.
 ```
-MONTE_CARLO_REGISTRY=$(aws ecr create-repository --repository-name monte-carlo-workshop | grep "repositoryUri" | awk '{split($0,a,":"); print a[2]}' | sed 's/"//g')
+MONTE_CARLO_REGISTRY=$(aws ecr create-repository --repository-name monte-carlo-workshop | jq --raw-output '.["repository"].repositoryUri')
```

Comment:
Just a note: it's not necessary to use jq for this, you can use --query parameter of the AWS CLI. This would have the same effect: 

aws ecr create-repository --repository-name monte-carlo-workshop --query repository.repositoryUri --output text